test/distributions/robust/arithmetic/collapsing/rankChangeAssign.chpl

```diff
@@ -0,0 +1,114 @@
+
+use Random;
+
+use driver;
+use driver_domains;
+use driver_arrays;
+
+config const seed = 1234;
+
+config const debug = false;
+
+proc DEBUG() { if debug then writeln(); }
+proc DEBUG(const args ...?n) {
+  if debug then writeln((...args));
+}
+
+test(DistType.default);
+test(DistType.block);
+test(DistType.cyclic);
```

Comment:
Just to make sure I"m understanding the high-level approach here:  This test explicitly tests default, block, and cyclic and then relies on this test being run against those same distributions (plus others... replicated?) as part of the robustness testing suite to get the cross product, is that right?

Reply:
Yes, it relies on different ``distType`` config param values being run in different testing configurations.